internal/super/flix.go

```diff
@@ -328,17 +377,37 @@ func getDeployedContracts(state *flowkit.State) flixkit.ContractInfos {
 			if _, ok := allContracts[name]; !ok {
 				allContracts[name] = make(flixkit.NetworkAddressMap)
 			}
+			if slices.Contains(excludeNetworks, network) {
+				continue
+			}
 			allContracts[name][network] = addr
 		}
 	}
 
+	// add contracts that have not been deployed
+	for _, c := range *state.Dependencies() {
```

Comment:
This should be `.Contracts` not `.Dependencies`. `Contracts` should include `Dependencies`, but not vice versa.

Reply:
flow.json

```
         "contracts": {
		"HelloWorld": "./cadence/contracts/HelloWorld.cdc"
	},
	"dependencies": {
		"Burner": {
			"source": "mainnet://f233dcee88fe0abe.Burner",
			"hash": "71af18e227984cd434a3ad00bb2f3618b76482842bae920ee55662c37c8bf331",
			"aliases": {
				"emulator": "f8d6e0586b0a20c7",
				"mainnet": "f233dcee88fe0abe",
				"testnet": "9a0766d93b6608b7"
			}
		},
...
```

Does .Contracts have addresses? Is Dependencies property exist in both .Contracts and .Dependencies?

Reply:
Dependencies are ingested and treated as contracts, so if you use `.Contracts` you get contracts and dependencies, but not the other way around.